uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/models/entitlement.model.js

```diff
@@ -69,6 +71,33 @@ define([
                 return instance.init(entitlement, true);
               });
             });
+        },
+
+        /**
+         * Returns all Leave Entitlement Log entries. Can filter by contact and
+         * period id.
+         *
+         * @param {Object} param - list of filters to pass to the api call.
+         * @return {Promise} - resolves to a list of entitlement log entries.
+         */
+        logs: function (params) {
+          params = _.clone(params || {});
```

Comment:
Is params object **always** flat? wouldn't it be better to use _.cloneDeep here?